controllers/sr_controller/sr_controller.py

```diff
@@ -150,62 +148,61 @@ def log_filename(zone_id: int) -> str:
     )
 
 
-class TeeStdout:
+class SimpleTee:
     """
-    Tee stdout also to the named log file.
+    Forwards calls from its `write` and `flush` methods to each of the given targets.
     """
 
-    def __init__(self, name: Path) -> None:
-        self.name = name
-
-    def __enter__(self):
-        self.stdout = sys.stdout
-        self.stack = contextlib.ExitStack()
-        self.file = self.stack.enter_context(open(str(self.name), mode='w'))
-        self.stack.enter_context(contextlib.redirect_stdout(self))  # type:ignore
-        self.stack.__enter__()
-
-    def __exit__(
-        self,
-        exctype: Optional[Type[BaseException]],
-        excinst: Optional[BaseException],
-        exctb: Optional[TracebackType],
-    ) -> None:
-        self.flush()
-        self.stack.__exit__(exctype, excinst, exctb)
```

Comment:
I notice you're no longer closing the handler on exit, is that intentional?

Obviously during cleanup it doesn't _really_ matter, as the process is dying anyway.

Reply:
Yes, see the docstring of the new function.

Essentially the issue boils down to what happens in the following code with regards to when the exception trace gets printed to stderr:
``` python
with foo():
    raise SomeError()
```
The steps being:
- `__enter__`
- `raise`
- `__exit__`
- traceback printed

Reply:
I suspect we might be able to clean up using an at-exit handler, but I don't think it's worth it.
The only thing of consequence which is left around here is the open file handle, which I don't think is an issue.

Reply:
Ah I see, nice idea!

It's only going to be open slightly longer than normal as the process cleans up and dies, which definitely isn't something we really care about.